packages/shared/package.json

```diff
@@ -2,6 +2,16 @@
 	"name": "@huggingface/shared",
 	"version": "0.0.1",
 	"description": "Shared utilities for Hugging Face's JavaScript libraries",
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"types": "./dist/index.d.ts",
+	"exports": {
+		".": {
+			"require": "./dist/index.js",
+			"import": "./dist/index.mjs",
+			"types": "./dist/index.d.ts"
+		}
+	},
```

Comment:
We don't need all this (not even the package.json) if we import the files directly (without using `@huggingface/shared`)